src/smclient/smclient.cpp

```diff
@@ -305,6 +305,14 @@ void SMClient::OnClockUnsynced()
     mCondVar.NotifyOne();
 }
 
+void SMClient::OnCertChanged(const iam::certhandler::CertInfo& info)
+{
+    LOG_DBG() << "Received on cert changed event: " << info;
```

Comment:
`Cert changed` or `Cert changed event received`.

Should be guarded with mutex.